persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -113,88 +125,53 @@ public void execute() {
                     + e.getMessage());
             return;
         }
-
+        if (schemaFiles.size() > 1) {
+            errStream.println("ERROR: persist directory contains multiple schema files. ");
```

Comment:
```suggestion
            errStream.println("ERROR: the persist directory allows only one model definition file, but contains many files.");
```

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)